packages/mdc-select/mdc-select.scss

```diff
@@ -27,12 +28,15 @@
 // postcss-bem-linter: define select
 .mdc-select {
   @include mdc-typography-base;
+  @include mdc-states;
+
```

Comment:
We might want to go ahead and remove the following `@each` which is applying letter-spacing typography styles, since IIUC we're probably removing that property from typography anyway, and this might be hard to catch when we do a sweep later... @lynnjepsen what do you think?